src/Plugin/DashboardBlockInterface.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Drupal\contacts\Plugin;
+
+/**
+ * Defines an interface for blocks placed on the Contacts Dashboard.
+ */
+interface DashboardBlockInterface {
+
+  const EDIT_LINK_TITLE = 'title';
+  const EDIT_LINK_CONTENT = 'content';
```

Comment:
Let's comment these while we're at it :)

Reply:
The NULL coalescing operator (??) was added in php7 (http://php.net/manual/en/migration70.new-features.php) so we should avoid using it in contrib if possible. Not to mention our dev server is 5.6 so breaks my environment :P

Reply:
https://www.drupal.org/project/drupal/issues/2842431 they are already planning on dropping 5.6 support. I'd be perfectly happy requiring php 7... Let's see what Jamie thinks...